src/ui/Tutorials/TutorialsSections/InternalTransfers.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import {TutorialSection} from './TutorialSection';
+import {Text} from '../../common/Text';
+import {SubTitle} from '../../common/SubTitle';
+import {SubSection} from '../../common/SubSection';
+import {PrismCode} from '../../Docs/PrismCode';
+import {Code} from '../../common/Code';
+import styled from 'styled-components';
+
+export const InternalTransfers = () => (
+  <>
+    <TutorialSection title="Internal Transfers">
+      <Text>
+        Internal transfers allow transfering amounts of your deversifi token balance to other users on the platform.
```

Comment:
transfering => transferring

`to other users on the platform` = `to other users of the protocol`